Build.ps1

```diff
@@ -3,27 +3,36 @@
 [System.Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSAvoidUsingPlainTextForPassword", "RegistryPassword")]
 
 param(
-    [Parameter(Mandatory = $false)]
+    [Parameter()]
     [ValidateNotNullOrEmpty()]
-    [string]$InstallSourcePath = (Join-Path $PSScriptRoot "\packages")
-    ,
+    [string]$InstallSourcePath = (Join-Path $PSScriptRoot "\packages"),
     [Parameter(Mandatory = $true)]
     [ValidateNotNullOrEmpty()]
-    [string]$SitecoreUsername
-    ,
+    [string]$SitecoreUsername,
     [Parameter(Mandatory = $true)]
     [ValidateNotNullOrEmpty()]
-    [string]$SitecorePassword
-    ,
-    [Parameter(Mandatory = $false)]
-    [string]$Registry = ""
-    ,
-    [Parameter(Mandatory = $false)]
-    [string]$RegistryUserName = ""
-    ,
-    [Parameter(Mandatory = $false)]
-    [string]$RegistryPassword = ""
-
+    [string]$SitecorePassword,
+    [Parameter()]
+    [string]$Registry = "",
+    [Parameter()]
+    [string]$RegistryUserName = "",
+    [Parameter()]
+    [string]$RegistryPassword = "",
+    [Parameter()]
+    [ValidateSet("9.3.0","9.2.0","9.1.1", "9.0.2")]
+    [string[]]$SitecoreVersion = @("9.3.0"),
+    [ValidateSet("xm","xp", "xc")]
+    [string[]]$Topology = @("xm","xp","xc"),
+    [ValidateSet("1909","1903","1809")]
+    [string[]]$WindowsVersion = @("1909"),
```

Comment:
default value of `WindowsVersion` needs to be `1809` (ltsc2019) so it builds the same images as today.

Reply:
BTW, it think is better to just call 1809 ltsc2019 in the paramters, since its what it's called in all tags...

Reply:
All good feedback @pbering . Making the changes now.